tests/unit/type/list.tcl

```diff
@@ -297,6 +405,30 @@ start_server {
         assert_error "WRONGTYPE*" {$rd read}
     }
 
+    test "BRPOPRPUSH with wrong source type" {
```

Comment:
there's really no value in adding these 3 tests.
although they don't take time, let's trim them.

Reply:
Done.